### PR TITLE
completed javascript-loops.

### PR DESCRIPTION
**Pull request by Dustin-Davis**
Screenshot 2021-12-08 172630

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ */
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = (0);
+  while (currentNumber < 10) {
+    currentNumber++;
+    number.push(currentNumber);
+  }
+  return number;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:17]: Counting should start from `1`:

---

image

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  return keys;
+}
+
+function getValues(object) {
+  var values = [];
+  for (var value in object) {
+    values.push(`${object[value]}`);
+  }
+  return values;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:68]: This should return an array of the actual values, not an array of their string versions:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-loops/main.js (hunk 2 above)
Comment (Dustin-Davis) [reply javascript-loops/main.js:68]: how would you fix that i am confused? or how would you go about fixing it?

**Comment by Dustin-Davis** [comment]
Screenshot 2021-12-08 180423

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+
+function getKeys(object) {
+  var keys = [];
+  for (var key in object) {
+    keys.push(`${key}`);
```
Comment (Zhiwutian) [inline javascript-loops/main.js:58]: you can just push the `key` variable here, it doesn't need to be converted to a string.

**Review by Zhiwutian — APPROVED** [review]
Approved with informational comment!

File: javascript-loops/main.js (hunk 3 above)
Comment (Dustin-Davis) [reply javascript-loops/main.js:58]: ok thank you for letting me know

**Pull request merged**